samples/apps/copilot-chat-app/copilotchat/Options/CopilotChatAIServiceOptions.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.ComponentModel.DataAnnotations;
+
+namespace SemanticKernel.CopilotChat.Config;
+
+/// <summary>
+/// Configuration options for Copilot Chat AI services, such as Azure OpenAI and OpenAI.
+/// </summary>
+public sealed class CopilotChatAIServiceOptions
```

Comment:
This is generic and re-usable.  Why not simply re-use from WebApi?

Reply:
I believe that would create a circular dependency.

Reply:
Right, but maybe we should extract the re-usable stuff to a lib. Maybe not in this PR, which is already quite big